dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/domain/SpanSerializerTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.datadog.android.tracing.internal.domain
+
+import com.datadog.android.utils.extension.assertMatches
+import com.datadog.android.utils.forge.Configurator
+import com.google.gson.JsonParser
+import datadog.opentracing.DDSpan
+import fr.xgouchet.elmyr.annotation.Forgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+
+@MockitoSettings(strictness = Strictness.STRICT_STUBS)
+@ForgeConfiguration(Configurator::class)
+internal class SpanSerializerTest {
+
+    lateinit var underTest: SpanSerializer
+
+    @BeforeEach
+    fun `set up`() {
+        underTest = SpanSerializer()
+    }
+
+    @Test
+    fun `it will serialize a span to it Json string representation`(@Forgery span: DDSpan) {
```

Comment:
I just wanted to make sure we are deserialising correctly because I was not sure if the deserialiser I used was the proper one for this class. If you don't mind I would keep this test for now as it might be handy during this trial step. On the other hand you always need a basic test for this Serialiser so why not having this, we have some logic inside that serialiser after all.